t/utils.t

```diff
@@ -0,0 +1,39 @@
+# t/utils.t
+#
+# Test LJ::Utils module
+#
+# This code was forked from the LiveJournal project owned and operated
```

Comment:
So since this is a net-new file, we should use the header that assigns copyright to Dreamwidth.

The LJ::Utils file I could go either way on... I don't know what we did with the LJ/User/FOOBAR.pm style modules, so I'd copy what we did there?

Reply:
done. the LJ/User/*.pm modules use the "forked from the livejournal project" headers so I kept that in Utils.pm too.